### DISCUSSION TITLE
FHIR R4-R5

### DISCUSSION THREAD

**Discussion by helloworldwecomeinpeace** [medplum/medplum#4103] [Q&A]
Hello, 

Im using Medplum with FHIR R4 to build an EMR. I tried to create a [chargeitem in R4 ](https://hl7.org/fhir/R4/chargeitem.html) against a [servicerequest](https://hl7.org/fhir/R4/servicerequest.html), but servicerequest is not related to chargeitem in R4.

But this is not the case for [chargeitem in R5](https://www.hl7.org/fhir/chargeitem.html). I am a new to FHIR hence I do not know how to proceed. Please help.

**Comment by codyebberson** [comment]
Hello @helloworldwecomeinpeace -

Please refer to the existing discussion on FHIR R5: https://github.com/medplum/medplum/discussions/2813

For linking a `ChargeItem` and `ServiceRequest` in FHIR R4, I would consider one of the following:

1. If you are using an `Encounter` with the `ServiceRequest`, then linking via `ChargeItem.context` (which is a `Reference` to the `Encounter`)
2. If not using an `Encounter`, then simply linking via `ChargeItem.supportingInformation` (which is a `Reference` to any resource type)
